test/cmd.js

```diff
@@ -14,12 +14,12 @@ var TEMP_DIR = path.resolve(__dirname, '..', 'temp', String(process.pid + Math.r
 
 describe('express(1)', function () {
   before(function (done) {
-    this.timeout(30000);
+    this.timeout(60000);
     cleanup(done);
   });
 
   after(function (done) {
-    this.timeout(30000);
+    this.timeout(120000);
```

Comment:
I remember that I raised the timeouts previously, because the AppVeyor tests would unpredictably timeout - if I recall the `--es2015` tests would vary greatly in duration - so bearing that in mind let me know if there's anything different that comes to mind that you'd like me to do. 

It looks like they passed this submission so maybe this issue has been resolved already.